docs/index.rst

```diff
@@ -10,24 +10,68 @@ SOSS Documentation
     :alt: eProsima
     :target: http://www.eprosima.com/
 
-:code:`soss` is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that
+*SOSS* is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that
 speak different languages.
 
 If one has, for instance, a pair of complex systems, A and B, and wills to combine them to create a larger, even more
-complex system, :code:`soss` can be chosen as an intermediate message-passing tool that, by speaking a common
+complex system, *SOSS* can be chosen as an intermediate message-passing tool that, by speaking a common
```

Comment:
I don't like the `can be chosen` expression. I'd change it to `allows` or something similar.

Reply:
ok, lo intento reformular.